contrib/builder/rpm/amd64/generate.sh

```diff
@@ -58,6 +61,10 @@ for version in "${versions[@]}"; do
 			# get rpm-build and curl packages and dependencies
 			echo 'RUN zypper --non-interactive install ca-certificates* curl gzip rpm-build' >> "$version/Dockerfile"
 			;;
+		photon:*)
+			echo "RUN ${installer} install -y wget git gzip rpm-build" >> "$version/Dockerfile"
+			echo "RUN wget -qO- https://gist.githubusercontent.com/toliaqat/686a7f05b0dcd006296a/raw/25592008aa215f6d8fe031536522315ac063f978/build-essential.txt | xargs ${installer} install -y" >> "$version/Dockerfile"
```

Comment:
yeah, I know :disappointed:, a metapackage is coming soon, but I thought I would get the ball rolling in the meantime.